modules/cnf-performing-end-to-end-tests-running-the-tests.adoc

```diff
@@ -10,34 +10,44 @@ Run the cluster latency tests to validate node tuning for your Cloud-native Netw
 
 [NOTE]
 ====
-When executing `podman` commands as a non-root or non-privileged user, mounting paths can fail with `permission denied` errors. To make the `podman` command work, append `:Z` to the volumes creation; for example, `-v $(pwd)/:/kubeconfig:Z`. This allows `podman` to do the proper SELinux relabeling.
+When executing `podman` commands as a non-root or non-privileged user, mounting paths can fail with `permission denied` errors. Depending on your local operating system and SELinux configuration, you might also experience issues running these commands from your home directory. To make the `podman` commands work, run the commands from a folder that is not your home directory, and append `:Z` to the volumes creation. For example, `-v $(pwd)/:/kubeconfig:Z`. This allows `podman` to do the proper SELinux relabeling.
```

Comment:
Hmm, :Z is to be selinux aware is fine, but this always worked for me from home dir. Do you have any link to an issue where using e.g. /tmp is needed?

Reply:
Thanks @MarSik for your feedback. No, I do not have any link to an issue where using e.g. /tmp is needed? 

@rohennes do you know of any such link?

Reply:
In a Slack chat with @MarSik, and feedback from @rohennes in the 'latency-tests-user-journal' Google Doc, it seems there is a use case - it is just /home/ronan that needs to be avoided, both /home/ronan/work and /tmp should work.

I will update the Note 

from 
To make the `podman` commands work, run the commands from a folder that is not your home directory, and append `:Z` to the volumes creation. 

to
To make the `podman` commands work, run the commands from a folder that is not your home/<username> directory, and append `:Z` to the volumes creation.

Reply:
@MarSik @rohennes the note has been reworded as above, and also updated where the note appears in the sections for the three individual tests - hwlatdetect, cyclictest, and oslat.